app/controllers/collections_controller.rb

```diff
@@ -132,6 +132,15 @@ def web_create_collection
     # Expose public licences and user created licences
     licence_table = Licence.arel_table
     @licences = Licence.where(licence_table[:private].eq(false).or(licence_table[:owner_id].eq(current_user.id)))
+    @collection_name = params[:collection_name]
+    @collection_title = params[:collection_title]
+    @collection_owner = params[:collection_owner]
+    @collection_abstract = params[:collection_abstract]
+    @licence_id = params[:licence_id]
+    @additional_metadata = []
+    unless params[:additional_key].nil? || params[:additional_key].nil?
```

Comment:
your checking the same thing on both sides of the OR?
params[:additional_key].nil?
params[:additional_key].nil?